pkg/runctl/controller.go

```diff
@@ -101,6 +129,10 @@ func (c *Controller) Run(threadiness int, stopCh <-chan struct{}) error {
 	if ok := cache.WaitForCacheSync(stopCh, c.pipelineRunSynced, c.tektonTaskRunsSynced); !ok {
 		return fmt.Errorf("failed to wait for caches to sync")
 	}
+
+	klog.V(2).Infof("Start metering: Create all %v minutes a new histogram", meteringHistogramInterval)
+	go c.meterCurrentPipelineStatus(meteringHistogramInterval)
```

Comment:
Better use `wait.Util()` (see some lines below) for repeated execution of the metering function, as it also reacts on `stopCh`.